packages/entando-apimanager/modules/api/apiManager.js

```diff
@@ -142,6 +118,94 @@ const normalizeErrorMessage = (message) => {
   return 'serverError';
 };
 
+export const authIntercept = () => {
+  if ('currentUser' in store.getState() && getRefreshToken()) {
+    authorize(getRefreshToken(), getAuthenticationToken());
+  }
+};
+
+const beginInterceptSetup = () => {
+  const configIntercept = {
+    // specify if request should be intercepted here we intercept
+    // intercepts only if refreshTokenConfig object props is specified
```

Comment:
no comments in the code, if you find them necessary it means the code is not readable

Reply:
okay. I will remove them